### PR TITLE
Releaser actions Provider Binaries

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: cloud/aws/.goreleaser.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
```
Comment (jyecusch) [inline cloud/aws/.goreleaser.yaml:3]: Are these comments left over from a template?

File: cloud/azure/.goreleaser.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
```
Comment (jyecusch) [inline cloud/azure/.goreleaser.yaml:3]: ?

File: cloud/gcp/.goreleaser.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
```
Comment (jyecusch) [inline cloud/gcp/.goreleaser.yaml:3]: ?

**Review by jyecusch — CHANGES_REQUESTED** [review]

File: cloud/aws/.goreleaser.yaml (hunk 1 above)
Comment (tjholm) [reply cloud/aws/.goreleaser.yaml]: this was copied over from the CLI, so will need to remove them there as well.

**Review by jyecusch — APPROVED** [review]
Just a lint issue to clean up then good to go

**Pull request merged**